File: eftl/sdk/dotnet/TIBCO.EFTL/json/JsonException.cs
/*
 * Copyright (c) 2001-$Date: 2016-03-11 16:29:10 -0800 (Fri, 11 Mar 2016) $ Cloud Software Group, Inc.
 * Licensed under a BSD-style license. Refer to [LICENSE]
 *
 * $Id: JsonException.java 84680 2016-03-12 00:29:10Z $
 */

using System;

public class JsonException : Exception
{

    public JsonException(string message) : base(message)
    {
    }

    public JsonException(string message, Exception inner) : base(message, inner)
    {
    }
}

File: eftl/sdk/java/src/com/tibco/eftl/json/JsonException.java
/*
 * Copyright (c) 2001-$Date$ Cloud Software Group, Inc.
 * Licensed under a BSD-style license. Refer to [LICENSE]
 *
 * $Id$
 */
package com.tibco.eftl.json;

public class JsonException extends RuntimeException {

    private static final long serialVersionUID = 1L;

    public JsonException(String message) {
        super(message);
    }

    public JsonException(Throwable cause) {
        super(cause);
    }
}
